ts/Core/Axis/Axis.ts

```diff
@@ -1930,6 +1930,7 @@ class Axis {
             tickPositionsOption = options.tickPositions,
             tickPositioner = options.tickPositioner,
             minorTickIntervalOption = this.getMinorTickInterval(),
+            minorTicksPerMajor = this.options.minorTicksPerMajor || 5,
```

Comment:
No need for a constant here, we're only using it once, so we can use it directly on line 1958. Also, no need for the `|| 5` part, if we make the option non-optional. Plus, no need for `this.options` because we already have the local `options` constant. So basically, remove line 1933 and change line 1958 to `this.tickInterval / options.minorTicksPerMajor`.